analysis/topeft_run2/analysis_processor.py

```diff
@@ -920,6 +936,16 @@ def process(self, events):
                                         if ((dense_axis_name in ["o0pt","b0pt","bl0pt"]) & ("CR" in ch_name)): continue
 
                                         hout[dense_axis_name].fill(**axes_fill_info_dict)
+                                        axes_fill_info_dict = {
+                                            dense_axis_name+"_sumw2" : dense_axis_vals[all_cuts_mask],
+                                            "channel"       : ch_name,
+                                            "appl"          : appl,
+                                            "process"       : histAxisName,
+                                            "systematic"    : wgt_fluct,
+                                            "weight"        : np.square(weights_flat),
```

Comment:
Right, it was easier to generalize, and let the datacard maker throw out the signal errors.

Reply:
Alternatively, we could check of the `self._wc_names_lst` is `None` and only fill the weights then.

Reply:
I like the idea of having a check on the `self._wc_names_lst` as it makes it more explicit that these `sumw2` histograms should only exist for the non-EFT samples

Reply:
I just remembered the whole point of this is we _need_ the MC errors from the SM point of our private samples, since that error goes into `fakes_sm`. I've restored it back to my original implementation.